styleguide/source/_patterns/03-organisms/wayfinder.twig

```diff
@@ -0,0 +1,22 @@
+<section class="ama_wayfinder">
+  <div class="ama_wayfinder--grid container">
+    <div class="ama_wayfinder--referrer">
+      {% if wayfinder.referred %}
+        {% set linkBackUrl = wayfinder.referred.referrerHref %}
+        {% set linkBackTitle = wayfinder.referred.referrerTitle %}
+        <div class="ama_wayfinder--referrer--link-back">
+          <a href="{{ linkBackUrl }}">
+            {% include '@atoms/media/icons/svg/icon-select.twig' %}
+            <span class="ama_wayfinder--referrer--link-back_text">
```

Comment:
This class should be `ama__wayfinder--referrer__link-back-text` if "link back text" is the name of the element or `ama__wayfinder--referrer__link-back__text` if "text" is a sub-element of "link back" (personally I would prefer not to have sub-elements of sub-elements).